src/App.jsx

```diff
@@ -9,32 +11,20 @@ export const tabs = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
-
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-          </li>
-
-          <li data-cy="Tab">
-            <a href="#tab-2" data-cy="TabLink">Tab 2</a>
-          </li>
+export const App = () => {
+  const [selectedTab, setSelectedTab] = useState(tabs[0]);
```

Comment:
no need to store the whole `tab` in the state. `id` is enough

Reply:
I have got mistake right here with test... 
<img width="440" alt="image" src="https://github.com/mate-academy/react_tabs-js/assets/101499979/f6055c9f-08c7-412b-bf3a-77e4a61d00a4">

Reply:
<img width="1107" alt="image" src="https://github.com/mate-academy/react_tabs-js/assets/101499979/05a3410b-fdec-4d48-8d65-2449f6723f9c">

Reply:
let's store the whole tab in state